### DISCUSSION TITLE
Customs Information -> Tariff Number

### DISCUSSION THREAD

**Discussion by salyangoz** [etsy/open-api#1285] [Q&A]
Is there any documentation or feature available that allows us to retrieve the "Customs information -> Tariff number" data from API services? This information is particularly required by companies when invoicing, as customs often request the tariff number. If we could retrieve it through the API, it would allow us to automatically include it in invoices.

![Screenshot 2024-08-20 at 15 09 48 (1)](https://github.com/user-attachments/assets/f30ea91a-6dee-4c41-a2d9-ded20e136394)

**Comment by amitkumar-dev92** [comment]
Hello @salyangoz 
currently there is no way to get/update tariff number via API.
